src/mongodb/schemas.ts

```diff
@@ -0,0 +1,58 @@
+import { prop, getModelForClass /*, ReturnModelType*/ } from "@typegoose/typegoose";
+
+export class User {
+  @prop({ required: true })
+  public id!: string;
+  @prop()
+  public email?: string;
+  @prop()
+  public login?: string;
+  @prop()
+  public password?: string;
+  @prop()
+  public passwordSalt?: string;
```

Comment:
Please use snake_case for the fields name since it's how they are written in the API Documentation.

Reply:
ESLint was screaming at me for doing that and it looks quite out of place in JS